src/BottomNavBar/getTabBarIcon.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { MaterialCommunityIcons, MaterialIcons } from '@expo/vector-icons';
+
+const getTabBarIcon = (navigation, focused, tintColor) => {
```

Comment:
This function is actually a component, you can call it `TabBarIcon`, it gets some value and returns a jsx element.